common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -207,6 +207,12 @@ object Mappings {
       "composerUrl" -> nonAnalyzedString
     )
 
+  val frontUsageMetadata =
+    nonDynamicObj(
+      "addedBy" -> nonAnalyzedString,
```

Comment:
Would a datetime be useful too?

Reply:
The time the image was used does get stored - just not in the metadata. It's handled like it for all other usages.